include/fdreadoutlibs/wib/TPEmulatorModel.hpp

```diff
@@ -286,6 +287,54 @@ class TPEmulatorModel : public readoutlibs::SourceEmulatorConcept
     }
     TLOG_DEBUG(TLVL_WORK_STEPS) << "Data generation thread " << m_this_link_number << " finished";
   }
```

Comment:
this huge block of commented code seems like bad practise to me.  Also, is this the *same* code as RAWWIBTriggerPrimitiveProcessor::unpack_tpframe_version_2() ?  (If so, should this code be in a separate class, for re-use where required ?)

Reply:
well spotted, thanks. I wanted to delete the commented out code but forgot before the PR was made.